module/VuFind/src/VuFind/Search/Base/HierarchicalFacetHelperInterface.php

```diff
@@ -44,24 +47,24 @@ interface HierarchicalFacetHelperInterface
      * Helper method for building hierarchical facets:
      * Sort a facet list according to the given sort order
      *
-     * @param array          $facetList Facet list returned from Solr
-     * @param boolean|string $order     Sort order:
+     * @param array            $facetList Facet list returned from Solr
+     * @param bool|string|null $order     Sort order:
      * - true|top  sort top level alphabetically and the rest by count
      * - false|all sort all levels alphabetically
      * - count     sort all levels by count
      *
      * @return void
      */
-    public function sortFacetList(&$facetList, $order = null);
+    public function sortFacetList(array &$facetList, bool|string|null $order = null): void;
 
     /**
      * Helper method for building hierarchical facets:
      * Convert facet list to a hierarchical array
      *
-     * @param string    $facet     Facet name
-     * @param array     $facetList Facet list
-     * @param UrlHelper $urlHelper Query URL helper for building facet URLs
-     * @param bool      $escape    Whether to escape URLs
+     * @param string              $facet     Facet name
+     * @param array               $facetList Facet list
+     * @param UrlQueryHelper|bool $urlHelper Query URL helper for building facet URLs
```

Comment:
In the long term, it seems like `?UrlQueryHelper` would make more sense as the type here, but it would require code changes to implement. Should we make a list of things to change in a second pass, or would it make sense to fix things like this in separate PRs and then reconcile those changes back here after they are merged?